src/components/Staff/UpdateStaffForTimeRangeForm.tsx

```diff
@@ -46,18 +46,25 @@ export const UpdateStaffForTimeRangeForm = ({
     setEndDate(date?.startOf('day') || moment());
   };
 
-  const handleStartTimeChange = (date: Moment | null) => {
-    setStartTime(date?.startOf('minute') || moment());
-    if (date && endTime && date.isAfter(endTime)) {
+  const handleStartTimeChange = (hour: number, minute: number) => {
+    const newStartTime = moment(startTime).set({hour, minute});
+    setStartTime(newStartTime);
+    if (newStartTime.isAfter(endTime)) {
```

Comment:
This needs to be before or equal. It's not valid to choose the same start and end time